index.js

```diff
@@ -31,6 +31,19 @@ exports.newProject = async function(options) {
   console.log('Copying project files complete.')
 };
 
+exports.reactProject = async function(options) {
```

Comment:
it's exact copy of `newProject`, better to pass what's different (`sourceDir`) as a parameter than copy whole method.

Reply:
since we are going to seperate repo, think of not doing that